ERROR	2022/10/22 21:41:30 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=492b54347eaa657e40ae76819a1fb10e268fae29d96cae537469782d2fa5d589&auth_timestamp=1666449687&auth_version=1.0&body_md5=9007316e3a40588592a34bf4dc0b5235": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:42:10 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=d8fb1c25a45b79df6fc54d365a3e0f353ebca36c9ffce85c430d4a94a5bfab2b&auth_timestamp=1666449728&auth_version=1.0&body_md5=28d91f3f836e96f2f446dbd25ba3a9ae": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:42:19 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=c59f4ee8c29a463c09e505eb1b53c73b3059908fa51c9155baf9149a8c9b4824&auth_timestamp=1666449737&auth_version=1.0&body_md5=eb17fd1e028cc9ee8f96f60aef2851a0": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:42:31 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=3e739f28c8807557666fc4be8a55883691387fb68d61c8175824af339599b8a4&auth_timestamp=1666449748&auth_version=1.0&body_md5=7112fe748e12897f248fbcf34af74653": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:42:35 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=f51c2211c2c63d9f65591045aa7379e12db57647702b650d0cb0e272436c1364&auth_timestamp=1666449753&auth_version=1.0&body_md5=b9ee0e9b3eb68559a627f254dade136b": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:29 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=cbe478335857beedb7934cd5fe297290e88c18b96a8484adaf0ea8772aac57b7&auth_timestamp=1666449867&auth_version=1.0&body_md5=5f743ab61b42fdb82df942929fcb921b": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:31 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=ccaf0d106a01ccfbdd39691514c8f03efa532d2f212782ce0b8e22dd1e005a7f&auth_timestamp=1666449869&auth_version=1.0&body_md5=4814d710f1b25b848273f131d3bedc81": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:34 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=62fd86030ae7789f6cb7e87512445a2d3af55cbde4dddea44976486709673bd3&auth_timestamp=1666449872&auth_version=1.0&body_md5=404d8f72a60cc1a037397359c67ca60c": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:36 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=2c3ca03a470bd852135ade8074a0c1f08cd07ea3c30194e72a3a03c417f72736&auth_timestamp=1666449874&auth_version=1.0&body_md5=ade38dd85f94431c795b7fd766e28e75": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:38 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=17924932a93ac13900001e5ab3211b704db19bd1e85f3b60fd46a1061b488cae&auth_timestamp=1666449876&auth_version=1.0&body_md5=404d8f72a60cc1a037397359c67ca60c": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:40 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=62ac9738f6e70fa50dd4299274f91b854477caa5de7d0c22d5dca19d3bfd3806&auth_timestamp=1666449878&auth_version=1.0&body_md5=acd28646e554ce7e1b3adaf75a503a8a": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:47 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=f63cedb658942fb5809db3782dac4b10c28731845d0a0c33476c81bd135802fd&auth_timestamp=1666449885&auth_version=1.0&body_md5=6de1bb2f6e580d613baf86c6924dd6df": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:49 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=0acc292abe80af0397c7359b6aee0887035f8d8dd240ba7b67ef2aacd405c4a8&auth_timestamp=1666449887&auth_version=1.0&body_md5=aaf385c95c17a7f809aeaa146f58f248": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:55 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=00667cd6f92088822211da71f9cf6f8d58b77cd2651852537383fe7ee5b06d5b&auth_timestamp=1666449893&auth_version=1.0&body_md5=d9277c2e58e61e1cba6e493d3c40b559": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:44:57 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=b5c5145818153b67a91291dc0f4fad09291b0573b9115640fcc9c838dca96c52&auth_timestamp=1666449895&auth_version=1.0&body_md5=522ee4bfbff5b97497e33a0343f6d295": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:31 start-monitoring.go:34: Post "http://:443/apps/9/events?auth_key=&auth_signature=1dac68602815444b78077760fcf2149ba406753faf4dd8ec66d1f027e22a6fc5&auth_timestamp=1666450709&auth_version=1.0&body_md5=3f6555ebd1c26fc42612f1cbfa1e4f0b": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:33 start-monitoring.go:94: Post "http://:443/apps/9/events?auth_key=&auth_signature=f8a5d50c75c007ccd2bc211a284043ee4a5324e15807f0857ffc35c59c0ad296&auth_timestamp=1666450711&auth_version=1.0&body_md5=68c6207285beaee4e1c9a6433b56edfa": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:35 start-monitoring.go:99: Post "http://:443/apps/9/events?auth_key=&auth_signature=742bd39b82c2b162bef7c9d61e7463dd777780fbb402939cc7002add3a9e438e&auth_timestamp=1666450713&auth_version=1.0&body_md5=ec638138cb027b055619360072cb7128": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:36 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=68016f6cbbe54732ad32d2ed0f1080676611f2b6760b96e8801a79292424c5d4&auth_timestamp=1666450714&auth_version=1.0&body_md5=d0c4f67a463c34a44f8f3293053aecf4": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:38 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=ce11f5c547fc038011faec45982cf6a9e0787e488120051198c926f30e0dc6bb&auth_timestamp=1666450716&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:41 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=4cf5e9109dcaa323dc2fbbdcc5b8d0daefcac2bc3f814d64170b0fa7e3eba310&auth_timestamp=1666450719&auth_version=1.0&body_md5=bdb3a6c25b3c3a0812423b25c608c38d": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:43 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=d613a878301db5250821312de1390250ab62f4fc91af8070401aedfabde22898&auth_timestamp=1666450721&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:46 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=2820d2a8c9278731f1d420487dbd9d149052ceec351043a56cfb68ebce1245b3&auth_timestamp=1666450724&auth_version=1.0&body_md5=d012a30ab0cbef0028681d77aeda5311": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:48 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=eded8de3b9851ed443cbf8c39af8ccdffdf5352071768b3f5e151e880db46143&auth_timestamp=1666450726&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:58:50 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=72c5cb256c17c87d3676138415fc59f6348f538a2a018304d180653c95e82a99&auth_timestamp=1666450728&auth_version=1.0&body_md5=d5ab7b74c7fdef72f919da6e0bd4ef44": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:59:43 start-monitoring.go:34: Post "http://:443/apps/9/events?auth_key=&auth_signature=e2b71fd7c3db36cc938af3ad966a9b8da769c85c2fc5a0a57130a17be6121b5b&auth_timestamp=1666450781&auth_version=1.0&body_md5=3f6555ebd1c26fc42612f1cbfa1e4f0b": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:59:45 start-monitoring.go:94: Post "http://:443/apps/9/events?auth_key=&auth_signature=e372de6215e4bf09a713015511feaf931bd1628ce2ebd055fab7e14773982cce&auth_timestamp=1666450783&auth_version=1.0&body_md5=1cad432c9861a48426775fff69d21d35": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:59:47 start-monitoring.go:99: Post "http://:443/apps/9/events?auth_key=&auth_signature=8093d8c4d73aed5ba35cb694b6d1ec489286cd5e71418505ea1e2380652b62a0&auth_timestamp=1666450785&auth_version=1.0&body_md5=05909c72ed76feb6e86946c299b89396": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:59:48 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=f4501ba4471979621f9da2f64da989140f656156f919f280e24d1a0c639dcf59&auth_timestamp=1666450786&auth_version=1.0&body_md5=3abbaae57771dc526482b63f62facc2d": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:59:50 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=d8c131d4889a4dfac23313bd7c6a805fedc6f4766d0b10aee706244c50772234&auth_timestamp=1666450788&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:59:53 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=588ccf697d88730c122db9dbc2c154c71b7dd18097034ee6dfe21bdbfb98e5ad&auth_timestamp=1666450791&auth_version=1.0&body_md5=35568ad35fd691e2ae4506492ddb0e2b": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:59:55 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=57d06888ea1aabbbb4e111891d3447a48e5a49da111de03174680646fab62316&auth_timestamp=1666450793&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 21:59:57 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=ffed49e07917a98e8f8e258ff080fe1d8436c632c55732839335c80313471467&auth_timestamp=1666450795&auth_version=1.0&body_md5=d454ff38335ef68b24c402aa507b03fd": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:00 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=61baf78aa906377856e78159787cf6630715a189f9c253e51edef292e3989195&auth_timestamp=1666450798&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:02 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=7e8bd7441f4ef289118ce3a2b1c0c04f595f39213586c7b7205f33615e3fdd92&auth_timestamp=1666450800&auth_version=1.0&body_md5=c83ece9e8c545203e6d5b0985fed09d4": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:04 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=1ea3637bb267eeba4391681e17d0ba200109d52063aa893984fc8789ef71c9a0&auth_timestamp=1666450802&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:07 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=c1464266ed77272685fa74b26abbc36331b5f40b8c1734eea1c479c708e695bc&auth_timestamp=1666450805&auth_version=1.0&body_md5=ba962684d0aa6a8227cb1c77bfffd649": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:52 start-monitoring.go:34: Post "http://:443/apps/9/events?auth_key=&auth_signature=d7766a65925dd93855fa1dd0bdb264f7df9293dc9d55ed2095d34a8d0e3b5a49&auth_timestamp=1666450850&auth_version=1.0&body_md5=3f6555ebd1c26fc42612f1cbfa1e4f0b": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:54 start-monitoring.go:94: Post "http://:443/apps/9/events?auth_key=&auth_signature=1f8793d500c2c1584beb17709cc640e783e1de87321476b04e72adaf0409e365&auth_timestamp=1666450852&auth_version=1.0&body_md5=2989685173b8117df96aaf3ae313fada": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:57 start-monitoring.go:99: Post "http://:443/apps/9/events?auth_key=&auth_signature=ed2a4ad34797db40dcd17e6e049752505849a9982f74d7789088fef87dac5010&auth_timestamp=1666450854&auth_version=1.0&body_md5=eda222269885dc1ce0324151037d046e": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:57 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=cd13a86280fdc8ab71c8c3126941bb4e32c4c124101c82301cf3c294918d1985&auth_timestamp=1666450855&auth_version=1.0&body_md5=6db8a9ad46fd87c5c11934341169d01f": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:00:59 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=9700007b9a6dec36ce9e52b94bea9c65ac0c490487ea81ddc595b69d7508e117&auth_timestamp=1666450857&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:01:02 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=40220a77d5d43f81be16d6f3f8c6a7faae238ebd47510aa9bb7ff0f11658561f&auth_timestamp=1666450860&auth_version=1.0&body_md5=a088a06611e6f4563f95e96bd5c10cd8": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:01:04 perform-checks.go:83: Post "http://:443/apps/9/events?auth_key=&auth_signature=a22b8b598bcfc76c95c0ede2bd1b3e2ff806d20515f1a2bf7b113bce75acfa93&auth_timestamp=1666450862&auth_version=1.0&body_md5=47df0b9468142a57862058e00fb765dc": dial tcp :443: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/10/22 22:16:34 perform-checks.go:83: Post "http://localhost:4001/apps/1/events?auth_key=abc123&auth_signature=a545162f88778d18dc1eb224736ff2b112d8ab6801eeda9aed9a97f41d794c18&auth_timestamp=1666451792&auth_version=1.0&body_md5=9f058ceeb0eb850d57f87fd5bac950de": dial tcp [::1]:4001: connectex: No connection could be made because the target machine actively refused it.
ERROR	2022/11/05 20:53:42 main.go:144: failed to connect to `host=localhost user=hadon database=`: failed SASL auth (FATAL: password authentication failed for user "hadon" (SQLSTATE 28P01))
ERROR	2022/11/06 21:37:58 main.go:138: failed to connect to `host=localhost user=hadon database=`: failed SASL auth (FATAL: password authentication failed for user "hadon" (SQLSTATE 28P01))
ERROR	2022/11/06 21:40:18 main.go:138: failed to connect to `host=localhost user=hadon database=`: failed SASL auth (FATAL: password authentication failed for user "hadon" (SQLSTATE 28P01))
